### ISSUE TITLE
Journal favicon setting not adapted during upgrade from OJS2 to OJS3

### ISSUE THREAD

**Issue by asmecher** [pkp/pkp-lib#4993]
See https://forum.pkp.sfu.ca/t/missed-favicons-during-migration-from-ojs2-to-ojs3/55187/3 for details.

The OJS 2.x journal setting for favicons appears to be called `journalFavicon`; the OJS 3.x setting appears to be called just `favicon`. Ensure that the OJS 2.x style is migrated to the OJS 3.x style during upgrade.

**Comment by Tribunal33** [comment]
@asmecher I would like to close this as won't fix?  @mfelczak do you happen to have journal from 2.x times, with a favicon, and can validate if this is still happening?

**Comment by asmecher** [comment]
Yup, calling this a WONTFIX. The OJS2 to OJS3 process is long mothballed.
